docs/docsite/rst/userguide/teams.rst

```diff
@@ -118,26 +126,22 @@ To add permissions to a Team:
 3. Click to select the resource to assign team roles and click **Next**.
 
 .. image:: ../common/images/teams-permissions-templates-select.png
+   :alt: Permissions Templates Select
 
 4. Click the checkbox beside the role to assign that role to your chosen type of resource. Different resources have different options available.
 
 .. image:: ../common/images/teams-permissions-template-roles.png
+   :alt: Permissions Template Roles
```

Comment:
I suggest something like:
"Add Teams Permissions Wizard step 3, choose the roles to apply to the previously selected resource."